e2e/cypress/tests/integration/account_settings/profile/profile_picture_spec.ts

```diff
@@ -10,6 +10,8 @@
 // Stage: @prod
 // Group: @account_setting
 
+import "cypress-file-upload";
```

Comment:
This is a temporary add until `tests/support/index.ts` is converted to Typescript: it already has this import, which exends cypress with `attachFiles`. Until then, the easiest way is to redeclare the import.